src/java.desktop/windows/classes/sun/awt/shell/Win32ShellFolder2.java

```diff
@@ -1192,16 +1189,12 @@ static Image getSystemIcon(SystemIcon iconType) {
      * Gets an icon from the Windows system icon list as an {@code Image}
      */
     static Image getShell32Icon(int iconID, int size) {
-        boolean useVGAColors = true; // Will be ignored on XP and later
         Toolkit toolkit = Toolkit.getDefaultToolkit();
         String shellIconBPP = (String)toolkit.getDesktopProperty("win.icon.shellIconBPP");
```

Comment:
Looks like these lines aren't used any more and should be removed too.

Reply:
Yes. Fixed.